linux_files/setup

```diff
@@ -5,7 +5,7 @@ CURDUR=""
 TMPDIR=""
 VERSION="1.1.25"
 GOVERSION="1.11.2"
-DOCKERVERSION="18.06.1-ce"
+DOCKERVERSION="18.09.0"
```

Comment:
I need to test everything before a version upgrade. Give me some time to check it